pandas/core/frame.py

```diff
@@ -1085,51 +1085,52 @@ def to_dict(self, orient='dict', into=dict):
 
         Returns
         -------
-        result : collections.Mapping like {column -> {index -> value}}
+        dict, list or collections.Mapping
+            like {column -> {index -> value}}
```

Comment:
Sorry to ask you one more change. Do you mind changing the description of the `Returns` to something more accurate? That is only true with the default `orient`. Also, it'd be nice to start it with a capital letter and end it with a period. Thanks!

Reply:
No problem, better make it right during this pull request than having to come back latter!